### PR TITLE
rename bearer to session

### PR DESCRIPTION
**Pull request by gr2m**
part of hoodiehq/hoodie#490

### CODE REVIEW

**Comment by HipsterBrown** [comment]
LGTM :+1:

File: routes/utils/request-to-bearer-token.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-module.exports = toBearerToken
-
-function toBearerToken (request) {
-  if (!request.headers.authorization) {
-    return ''
-  }
-  return request.headers.authorization.substr(7)
```
Comment (jameswestnz) [inline routes/utils/request-to-bearer-token.js]: This makes me wonder... should the prefix for this header be set in the config? Means the code for both `server` and `client` could be automated to some extent as the config would be common across both.

Comment (gr2m) [inline routes/utils/request-to-bearer-token.js]: I wouldn’t make it configurable, but we could make it more clear with something like `'session '.length`

Comment (jameswestnz) [inline routes/utils/request-to-bearer-token.js]: Fair point - based on this it's not major, as a user would still be able to see what's going on when the search for the word `session` in this repo, as the function contains `session` :)

**Comment by jameswestnz** [comment]
Other than the possibility of uppercasing the s in `session`, LGTM 👍

**Comment by Charlotteis** [comment]
LGTM :)

**Comment by Charlotteis** [comment]
LGTM :) But I think we should change `session` to `Session`

**Pull request merged**